adapters/uniqueid_linux.c

```diff
@@ -4,14 +4,28 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <string.h>
-#include <uuid/uuid.h>
+#include <sys/random.h>
 
 #include "azure_macro_utils/macro_utils.h"
 #include "azure_c_shared_utility/uniqueid.h"
 #include "azure_c_shared_utility/xlogging.h"
 
 MU_DEFINE_ENUM_STRINGS(UNIQUEID_RESULT, UNIQUEID_RESULT_VALUES);
 
+// UUID fields as specified in RFC 4122
+struct uuid {
```

Comment:
Can we define struct in sync with other code in sdk?

Something like this:

typedef struct UUID_TAG
{
    uint32_t    time_low;
    uint16_t    time_mid;
    uint16_t    time_hi_and_version;
    uint16_t    clock_seq_and_variant;
    uint8_t      node[6];
} UUID;

https://github.com/Azure/azure-c-shared-utility/blob/b38adefd7d0f94d4118a1dc80b285f571a6bb8e1/adapters/socketio_mbed.c#L26